components/Block/RichText.vue

```diff
@@ -0,0 +1,19 @@
+<script setup lang="ts">
+import type { BlockProps } from './types';
+
+const { $directus, $readItem } = useNuxtApp();
+
+const props = defineProps<BlockProps>();
+
+const { data: comp } = useAsyncData(() =>
```

Comment:
```suggestion
const { data: comp } = useAsyncData(props.uuid, () =>
```

otherwise Nuxt's caching won't be able to differentiate between uses of this block